themes/gatsby-starter-theme-blog-mdx/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "gatsby-starter-theme-blog-mdx",
```

Comment:
Hate to nit on naming, but should this be gatsby-theme-blog-mdx or gatsby-starter-theme-blog-mdx? The latter seems a little confusing!

Reply:
I really have no idea how to name this sort of thing – more than happy to change this to whatever makes most sense. To clarify, this is meant to be a starter that uses the `gatsby-theme-blog-mdx` theme

Reply:
`gatsby-theme-blog-mdx`, but we're going to make the `-blog` named one MDX based anyway so the naming will change.

Reply:
the two blog themes (mdx+remark) will merge into one with the data theme (this is part of what I was experimenting with on my own blog), so the starter will eventually replace https://github.com/gatsbyjs/gatsby-starter-blog. As such, it's a temporary thing in this directory and the naming probably doesn't matter much.

Reply:
About what I expected -- thanks!